Nos/Service/Relay/RelayService.swift

```diff
@@ -709,8 +710,6 @@ extension RelayService: WebSocketDelegate {
             switch event {
             case .connected:
                 await handleConnection(from: client)
-            case .viabilityChanged(let isViable) where isViable:
-                await handleConnection(from: client)
```

Comment:
I'm not 100% sure but would we want to keep the `case` and `Log.debug("viabilityChanged")` or `Log.info` here so this is visible to us in the console? Maybe that would help us to determine whether we need `.viabilityChanged` or not. But maybe it wouldn't -- I'm not really sure.

Reply:
It looks to me like most of the relays are calling `.viabilityChanged(true)` right after connection, so I'm not sure it would be useful. What I am curious about is how often `.viabilityChanged(false)` is actually used in the wild. I haven't seen it in my testing. If we wanted to track that I would suggest adding an analytics event for it, although I don't know if we would do anything differently if it did happen often... so I'm not that concerned either way.